extensions/objects/templates/sanitized-url-editor.component.ts

```diff
@@ -21,9 +21,14 @@ import { downgradeComponent } from '@angular/upgrade/static';
 
 interface SanitizedUrlSchema {
   type: string;
-  validators: {
+  validators: [{
     id: string;
-  }[];
+  },
+    {
+      id: string;
+      regexPattern: string;
+    }
+  ];
```

Comment:
Let's do
```
validators: [
  {
    id: string;
  }, 
  {
    id: string;
    regexPattern: string;
  }
]

Reply:
@vojtechjelinek I pushed this change and also removed that extra space you pointed out in your comment below.